ui/common/css/header/_header.scss

```diff
@@ -3,7 +3,14 @@
 @import 'title';
 @import 'buttons';
 
-#top {
+body {
+  --dropdown-top: 40px;
+  @include breakpoint($mq-site-header-tall) {
+    --dropdown-top: 60px;
```

Comment:
why hardcode these? They exist as variables `$site-header-short-height` & `$site-header-tall-height`

Reply:
that's your code.